packages/mdc-shape/mdc-shape.scss

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+.mdc-shape-angled-corner-container {
```

Comment:
Should we add `box-sizing: border-box` to this class and the `__corner` class?

We've previously talked about specifying `box-sizing` on every single CSS class to avoid unexpected bugs.

Reply:
These two classes don't have any width or padding styles, so setting box-sizing one way or the other has no effect (I tested it, screenshot diff and all). Do you still think we should add it regardless?

Reply:
After talking offline, no, I don't think we should add `box-sizing` here. YAGNI :grin: